tests/jest/config/platform-properties.json

```diff
@@ -132,7 +132,7 @@
   "ansible.credential.selection.title": "Credential",
   "ansible.credential.selection.placeholder": "Select Ansible credential",
   "ansible.no.credential": "Ansible credential is required to create an Ansible job. Create your console by clicking the following link:",

```

Comment:
Should this say "Create your **credential** by clicking the following link:"?

Reply:
Thanks for catching up!

Reply:
Sure thing! Looks like there are some leftover Git markings here from the change...